example/ctc/README.md

```diff
@@ -49,68 +55,26 @@ def lstm_unroll(num_lstm_layer, seq_len,
-2017-07-08 13:25:04,041 Epoch[96] Batch [150]	Speed: 4247.44 samples/sec	Accuracy=0.831855
-2017-07-08 13:25:16,058 Epoch[96] Batch [200]	Speed: 4260.77 samples/sec	Accuracy=0.830840
-2017-07-08 13:25:28,109 Epoch[96] Batch [250]	Speed: 4248.44 samples/sec	Accuracy=0.827168
-2017-07-08 13:25:40,057 Epoch[96] Batch [300]	Speed: 4285.23 samples/sec	Accuracy=0.832715
-2017-07-08 13:25:40,782 Epoch[96] Train-Accuracy=0.830729
-2017-07-08 13:25:40,782 Epoch[96] Time cost=73.098
-2017-07-08 13:25:40,821 Saved checkpoint to "ocr-0097.params"
-2017-07-08 13:25:45,886 Epoch[96] Validation-Accuracy=0.840820
-2017-07-08 13:25:59,283 Epoch[97] Batch [50]	Speed: 4271.85 samples/sec	Accuracy=0.831648
-2017-07-08 13:26:11,243 Epoch[97] Batch [100]	Speed: 4280.89 samples/sec	Accuracy=0.835371
-2017-07-08 13:26:23,263 Epoch[97] Batch [150]	Speed: 4259.89 samples/sec	Accuracy=0.831094
-2017-07-08 13:26:35,230 Epoch[97] Batch [200]	Speed: 4278.40 samples/sec	Accuracy=0.827129
-2017-07-08 13:26:47,199 Epoch[97] Batch [250]	Speed: 4277.77 samples/sec	Accuracy=0.834258
-2017-07-08 13:26:59,257 Epoch[97] Batch [300]	Speed: 4245.93 samples/sec	Accuracy=0.833770
-2017-07-08 13:26:59,971 Epoch[97] Train-Accuracy=0.844727
-2017-07-08 13:26:59,971 Epoch[97] Time cost=72.908
-2017-07-08 13:27:00,020 Saved checkpoint to "ocr-0098.params"
-2017-07-08 13:27:05,130 Epoch[97] Validation-Accuracy=0.827962
-2017-07-08 13:27:18,521 Epoch[98] Batch [50]	Speed: 4281.06 samples/sec	Accuracy=0.834118
-2017-07-08 13:27:30,537 Epoch[98] Batch [100]	Speed: 4261.20 samples/sec	Accuracy=0.835352
-2017-07-08 13:27:42,542 Epoch[98] Batch [150]	Speed: 4264.88 samples/sec	Accuracy=0.839395
-2017-07-08 13:27:54,544 Epoch[98] Batch [200]	Speed: 4266.31 samples/sec	Accuracy=0.836328
-2017-07-08 13:28:06,550 Epoch[98] Batch [250]	Speed: 4264.50 samples/sec	Accuracy=0.841465
-2017-07-08 13:28:18,622 Epoch[98] Batch [300]	Speed: 4241.11 samples/sec	Accuracy=0.831680
-2017-07-08 13:28:19,349 Epoch[98] Train-Accuracy=0.833984
-2017-07-08 13:28:19,349 Epoch[98] Time cost=73.018
-2017-07-08 13:28:19,393 Saved checkpoint to "ocr-0099.params"
-2017-07-08 13:28:24,472 Epoch[98] Validation-Accuracy=0.818034
-2017-07-08 13:28:37,961 Epoch[99] Batch [50]	Speed: 4242.14 samples/sec	Accuracy=0.835861
-2017-07-08 13:28:50,031 Epoch[99] Batch [100]	Speed: 4241.94 samples/sec	Accuracy=0.846543
-2017-07-08 13:29:02,108 Epoch[99] Batch [150]	Speed: 4239.22 samples/sec	Accuracy=0.850645
-2017-07-08 13:29:14,160 Epoch[99] Batch [200]	Speed: 4248.34 samples/sec	Accuracy=0.844141
-2017-07-08 13:29:26,225 Epoch[99] Batch [250]	Speed: 4243.71 samples/sec	Accuracy=0.842129
-2017-07-08 13:29:38,277 Epoch[99] Batch [300]	Speed: 4248.07 samples/sec	Accuracy=0.851250
-2017-07-08 13:29:38,975 Epoch[99] Train-Accuracy=0.854492
-2017-07-08 13:29:38,976 Epoch[99] Time cost=73.315
-2017-07-08 13:29:39,023 Saved checkpoint to "ocr-0100.params"
-2017-07-08 13:29:44,110 Epoch[99] Validation-Accuracy=0.851969
```

Comment:
The expected accuracy is important information to validate if the example works. Can we either keep it in readme, or add assertion in the code to ensure the example actually learns?

Reply:
@safrooze will be following this with another PR end of week... he can address additional updates with his revision.